dissect/target/loaders/log.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from dissect.target import Target
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.loader import Loader
+
+
+class LogLoader(Loader):
+    LOGS_DIRS = {
+        "EvtxPlugin": "sysvol/windows/system32/winevt/logs",
+        "EvtPlugin": "sysvol/windows/system32/config",
+    }
+
+    @staticmethod
+    def detect(path: Path) -> bool:
+        return False
```

Comment:
This could check for file or directory containing `*.evtx` or `*.evt` files perhaps.

Reply:
Sounds reasonable. We can always reevaluate in the future if we see a need for it.

Regarding `rglob`, I don't think `rglob` should be allowed here, only a `glob` of the current path (which must then of course be a directory, or a single file matching the glob pattern).